src/SFA.DAS.Employer.PR.Web/Extensions/ClaimsPrincipalExtensions.cs

```
@@ -22,6 +23,50 @@ private static Guid GetClaimValue(ClaimsPrincipal principal, string claimType)
     public static EmployerIdentifier GetEmployerAccount(this ClaimsPrincipal user, string employerAccountId)
         => GetEmployerAccounts(user)[employerAccountId.ToUpper()];
 
+    public static bool CheckUserAccountAccess(this ClaimsPrincipal user, string employerAccountId, EmployerUserRole userRoleRequired)
+    {
+        var employerAccountClaim = user.FindFirst(c => c.Type.Equals(EmployerClaims.AccountsClaimsTypeIdentifier));
+
+        if (employerAccountClaim?.Value == null)
+            return false;
+
+        EmployerUserAccountItem? employerAccount = null;
+        try
```

What is the point of catching exception, just let it bubble up, as the expectations is to have serialised content in there.

---

removed the try catch